Assignment1.py

```diff
@@ -45,6 +53,7 @@
 r2 = pipeline.score(X_test_sc, y_test)
 print(f"RFR: {r2}")
 
+#Other models - SVC and Logistic Regression
 svclassifier = SVC(kernel='poly', degree=8)
 svclassifier.fit(X_train_sc, y_train)
 
```

Comment:
Again the code runs fine here and everything ran right, but print out the values you get from the classifier.fit so we can see if the models are accurate or not

Reply:
Well the code includes confusion matrix for both models which shows accuracy, precision, recall and F1 score.